action.yaml

```diff
@@ -10,17 +13,25 @@ inputs:
   patched-tag:
     description: 'The new patched image tag'
     required: true
+  timeout:
+    description: 'The timeout for the copa action'
+    default: '5m'
   buildkit-version:
     description: "Buildkit version to use with Copa"
   copa-version:
     description: "Copa version to use"
+  output:
+    description: "Output filename"
+  format:
+    description: "Output format"
+    default: "openvex"
```

Comment:
to be consistent I think we should also set a generic default output file name

Reply:
actually, I think we should remove the defaults in the case that someone is using an older version of copa

Reply:
> actually, I think we should remove the defaults in the case that someone is using an older version of copa

I agree but technically the format default is considered only if output is specified, see https://github.com/Vombato/copa-action/blob/042b66007c6bba597b8bd403a30178b0093321f7/entrypoint.sh#L15

In any case, your choice 🙌🏻